cmd/npmExecuteLint.go

```diff
@@ -62,7 +63,7 @@ func (u *lintUtilsBundle) getGeneralPurposeConfig(configURL string) {
 
 	err = u.FileWrite(filepath.Join(".pipeline", ".eslintrc.json"), content, os.ModePerm)
 	if err != nil {
-		log.Entry().Warnf("failed to write .eslintrc.json file to .pipeline/: %w", err)
+		//log.Entry().Warnf("failed to write .eslintrc.json file to .pipeline/: %w", err)
```

Comment:
In the end, this file should not be part of the PR. Do you still receive the errors if you revert this file?

Reply:
Fixed with an already mergen PR [link](https://github.com/SAP/jenkins-library/pull/1774)